site/viamguides/src/control-module/control-module.md

```diff
@@ -0,0 +1,577 @@
+author: Joyce
+id: control-module
+summary: Add control logic to your machine to run in a loop in the background
+categories: Getting-Started, Developer
+environments: web
+status: Published
+feedback link: https://github.com/viam-devrel/viamcodelabs/issues
+tags: Getting Started, Developer
+
+# Build a Viam module for control logic
+
+<!-- ------------------------ -->
+
+## Overview
+
+Duration: 1
+
+Let’s say you have a machine in Viam with a sensor and an actuator, like a webcam and a lamp plugged into a smart outlet. You want the machine to monitor the video feed, in this case, turning on the lamp when a person is detected.
```

Comment:
Maybe we should add some context about why running it on the device versus some external environment is important for this use case. You could have a script that does this automation running on your laptop, in the cloud, or some web app as well.